src/benchmark.cc

```diff
@@ -414,7 +418,8 @@ bool BenchmarkFamilies::FindBenchmarks(
           instance.name += StringPrintF("/threads:%d", instance.threads);
         }
 
-        benchmarks->push_back(instance);
+        if (re.Match(instance.name))
```

Comment:
it's not required, but for consistency these should have {} even though they're single-line.